app/build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
     })
     compile 'com.android.support:appcompat-v7:25.4.0'
     compile 'com.android.support.constraint:constraint-layout:1.0.0-beta4'
+    compile 'com.android.support.constraint:constraint-layout:1.0.2'
```

Comment:
You might want to remove the `compile 'com.android.support.constraint:constraint-layout:1.0.0-beta4'`